connector/mqtt/vernemq/v2k-brigde/README.md

```diff
@@ -0,0 +1,36 @@
+# **V2K-bridge**
+The V2K-bridge service implements a bridge between VerneMQ broker and Kafka broker. Basically, it subscribes to some MQTT topics and forwards the messages to some Kafka topics following the dojot's topics rules. In order to scale the bridge service, shared MQTT subscriptions are used, which allows to instantiate a group of consumers, i.e. a group of bridge instances, when necessary. The communication between the bridge and the VerneMQ is secured with mutual TLS. Soon, the communication with Kafka will also use mutual TLS.
+
+
+<p align="center"> 
+<img src="./doc/diagrams/bridge.png">
+</p>
+
+# **Configuration**
+
+## **Environment Variables**
+
+Before running the kubernetes environment with the V2K-bridge service, make sure you configure the environment variables to match your needs. The variables can be configured in the .yaml file of the service.
+
+Key                      | Purpose                                                             | Default Value       | Valid Values   |
+------------------------ | ------------------------------------------------------------------- | ---------------     | -------------- |
+HOSTNAME                 | Hostname to be used in the certificate common name                  | broker              | hostname/IP    |
```

Comment:
I think HOSTNAME is an environment default for k8s, so maybe we shouldn't overwrite that.